### DISCUSSION TITLE
help me Encountered When I was digging chia, a RuntimeError: Unknown exception error occurred, which caused my drawing progress to be stuck in %1. There is no next step, and the farm does not display any images. How can I solve this problem? Is anyone else digging? Thank you very much

### DISCUSSION THREAD

**Discussion by jy03677619** [Chia-Network/chia-blockchain#2057] [Ideas]
When I was digging chia, a RuntimeError: Unknown exception error occurred, which caused my drawing progress to be stuck in %1. There is no next step, and the farm does not display any images. How can I solve this problem? Is anyone else digging?

**Comment by hoffmang9** [comment]
This is almost always a problem with RAM.
